pkg/translate/translate_test.go

```diff
@@ -198,3 +198,49 @@ func errToString(err error) string {
 	}
 	return err.Error()
 }
+
+func TestNewTranslator(t *testing.T) {
+	tests := []struct {
+		name         string
+		minorVersion version.MinorVersion
+		wantVer      string
+		wantErr      bool
+	}{
+		{
+			name:         "version 1.3",
+			minorVersion: version.NewMinorVersion(1, 3),
+			wantVer:      "1.3",
+			wantErr:      false,
+		},
+		{
+			name:         "version 1.4",
+			minorVersion: version.NewMinorVersion(1, 4),
+			wantVer:      "1.4",
+			wantErr:      false,
+		},
+		{
+			name:         "version 1.5",
```

Comment:
I assume this test will break when 1.5 is added? If so, there should be a comment to that effect.

Reply:
yes, will create a new PR to address all comments in the cherrypick.